lib/controller.js

```diff
@@ -62,7 +62,7 @@ Tessel.list = function(opts) {
       // If there were no Tessels found
       if (foundTessels.length === 0) {
         // Report the sadness
-        reject(noTessels);
+        return reject(noTessels);
```

Comment:
Shouldn't change anything because this is an if-else block with nothing after it but it's good to have.